src/datastructure.h

```diff
@@ -97,9 +97,10 @@ typedef struct {
 	unsigned char magic;
 	int count;
 	int blockedcount;
+	uint32_t domainhash;
 	size_t domainpos;
 } domainsData;
-ASSERT_SIZEOF(domainsData, 24, 16, 16);
+ASSERT_SIZEOF(domainsData, 32, 20, 20);
```

Comment:
As I said above, it should be `24` for 64 bit because the variable fits into already present padding space.

Reply:
Sorry completely skipped over changing that.  I've committed a change with `24, 20, 20`